docs/index.md

```diff
@@ -1093,10 +1093,26 @@ Can be specified multiple times.
 
 Files having this extension will be considered test files. Defaults to `js`.
 
-Affects `--watch` behavior.
-
 Specifying `--extension` will _remove_ `.js` as a test file extension; use `--extension js` to re-add it. For example, to load `.mjs` and `.js` test files, you must supply `--extension mjs --extension js`.
 
+### `--watch-files <file|directory|glob>`
+
+> _New in v6.3.0_
+
+List of comma separated paths or globs to watch when `--watch` is set. If a file matching the given If the argument is a glob a change to any file matching the glob will rerun the tests.
+
+If the path is a directory all files and subdirectories will be watched.
+
+By default all files in the current directory having on of the extensions provided by `--extensions` and not contained in the `node_modules` or `.git` folders are watched.
```

Comment:
[...] having **one** of the extensions [...]
The option is `--extension` instead of `--extensions`.

Reply:
https://github.com/mochajs/mocha/pull/3980/commits/ba7667e29c72589ec50ed00411fe272b3071808b